msteams-platform/tabs/quickstarts/create-channel-group-tab-yeoman.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Quickstart: Create a Custom channel and group Tab with Node.js and the Teams Yeoman Generator"
+author: laujan
+description: A quickstart guide to creating a channel and group tab with the Teams Yeoman Generator.
+ms.topic: quickstart
+ms.author: laujan
+---
+# Quickstart: Create a custom channel and group tab with Node.js and the Teams Yeoman Generator 
+
+>[!NOTE]
+>This quickstart follows the steps outlined in the [Build Your First Microsoft Teams App](/OfficeDev/generator-teams/wiki/Build-Your-First-Microsoft-Teams-App) Wiki found in the Microsoft OfficeDev GitHub repository.
```

Comment:
I think this is a link to a github repository? If so, you need to include the entire https://github.com portion. Only URLs prefaced with https://docs.microsoft.com get the relative link treatment.